awx/main/scheduler/task_manager.py

```diff
@@ -68,7 +68,7 @@ def after_lock_init(self):
         '''
         Init AFTER we know this instance of the task manager will run because the lock is acquired.
         '''
-        instances = Instance.objects.filter(~Q(hostname=None), capacity__gt=0, enabled=True)
+        instances = Instance.objects.filter(~Q(hostname=None), enabled=True)
```

Comment:
I had to change this so that the code elsewhere that "picks the execution node" would *consider* capacity=0 instances for project update runs (which now require 0 capacity when `IS_K8S=True`).

I probably want to run this through a SLOWYO to make sure nothing breaks here because we're now *technically* tracking instances with capacity=0 in this data structure.  I'd *expect* the other methods later on that "find the best match with enough capacity", but it would be good to run this through integration and make sure we don't see anything concerning in our tests that exercise instance group selection and capacity.